homeassistant/components/switchbot/sensor.py

```diff
@@ -33,6 +33,13 @@
         entity_registry_enabled_default=False,
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
+    "wifi_rssi": SensorEntityDescription(
+        key="wifi_rssi",
+        native_unit_of_measurement=SIGNAL_STRENGTH_DECIBELS_MILLIWATT,
+        device_class=SensorDeviceClass.SIGNAL_STRENGTH,
+        entity_registry_enabled_default=False,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
```

Comment:
`self._attr_name = f"{switchbot_name} {sensor.title()}"`
should probably change to
`self._attr_name = f"{switchbot_name} {sensor.replace('_',' ').title()}"`